packages/cdk-api-gateway-domain-multi-region/lib/alias-target.ts

```diff
@@ -0,0 +1,11 @@
+interface GetAliasTargetProps {
+    companyDomainName: string;
+    project: string;
+    baseEnv: string;
+    dynamicEnv?: string;
+}
+export function getAliasTarget(props: GetAliasTargetProps) {
```

Comment:
Would it make sense for other constructs to import this in and use it? It seems like a function that would be used all over for route 53 stuffs.